### PR TITLE
feat(sources): add pagination to the table

### LINKED ISSUE #694: Sources: Add pagination
## User Story
As a user, I want to set the amount of items to display per page and to go to pages.

## Impacts
UI

## Role
Any user that has permissions to look at the sources

## Assumptions
None

## Acceptance Criteria

- [ ] Be able to change number of items to display per page
- [ ] Read results from other pages

### PR DESCRIPTION
**Pull request by boaz0**
closes #694

### CODE REVIEW

File: src/pages/sourceSettings/sourceSettings.tsx [hunk 1]
```diff
@@ -20,41 +20,72 @@ import Header from './header';
...
+
+  public onSearch(searchQuery) {
+    const newQuery = { ...this.props.query, ...searchQuery };
+    const res = Object.keys(newQuery)
+      .filter(k => newQuery[k])
+      .reduce((acc, curr) => {
+        const currQuery = `${curr}=${newQuery[curr]}`;
+        return acc === null ? currQuery : `${acc}&${currQuery}`;
+      }, null);
+    this.props.fetch(res);
```
Comment (dlabrecq) [inline src/pages/sourceSettings/sourceSettings.tsx]: Is there any help text to suggest that the filter must be an exact match?

When filtering by type, I expect to see a filter results row and filter chips, similar to the details pages. 

I also expect this to be a partial, logical AND filter. For example, users should be able to enter "Red" and see all partial matches. However, I only see an unexpected error.

Screen Shot 2019-05-06 at 11 25 54 AM

FYI, I cannot get out of this state, unless I delete the filter text and hit enter.

Comment (boaz0) [reply src/pages/sourceSettings/sourceSettings.tsx]: This addresses only #694, I can add filtering but the amount of files and code to review will be much bigger.

If that's not a problem on your side, let me know so I will continue working on this.

Comment (boaz0) [reply src/pages/sourceSettings/sourceSettings.tsx]: In addition, the error you get is irrelevant to `exact match`. 
The backend will throw an error if the type filter is not one of ["OCP", "AWS'].
I suggest to replace it to selector when `Type`

Comment (dlabrecq) [reply src/pages/sourceSettings/sourceSettings.tsx]: We can merge as is and address #649 separately. I was just confused by the filtering changes here.

**Comment by boaz0** [comment]
Thanks @dlabrecq

**Pull request merged**